packages/atomic/src/components/common/atomic-modal/atomic-modal.tsx

```diff
@@ -130,67 +135,75 @@ export class AtomicModal implements InitializableComponent<AnyBindings> {
   public render() {
     this.updateBreakpoints();
 
+    const content = (
```

Comment:
Nitpick, but I generally prefer callbacks when separating parts of the state, even when we're rendering them either way.

Something like:
```tsx
const Content = () => ...;
...
return <something><Content /></something>
```

Since you're in a hurry to merge, it's not really a big deal.

Reply:
Did it!